src/utils/constants.js

```diff
@@ -0,0 +1,18 @@
+export const BASE_URL = 'http://localhost:3000';
```

Comment:
`export const BASE_URL = 'http://localhost:3000';`

I can't sign up because requests go to `localhost`. See here http://joxi.ru/bmoKL1Vu7KoEGm . You need to change the url